frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/TagRow.tsx

```diff
@@ -99,80 +96,77 @@ export const TagRow = ({ feature }: IFeatureOverviewSidePanelTagsProps) => {
         }
     };
 
+    const addTagButton = (
+        <StyledAddTagButton
+            size='small'
+            permission={UPDATE_FEATURE}
+            projectId={feature.project}
+            variant='text'
+            onClick={() => {
+                setManageTagsOpen(true);
+            }}
+        >
+            <StyledAddIcon /> Add tag
```

Comment:
Isn't there typically a prop to add icons to buttons? 
Should this be something like:

```suggestion
            }}
            startIcon={<StyledAddIcon />}
        >
            Add tag
```

I know the permission button is a custom component, but I think it inherits most of the button props. And if it doesn't inherit this one, maybe it should? Not really important, though, as MUI icons typically get all the needed stuff to be hidden for screen readers etc anyway.